### ISSUE TITLE
Casino challenge's Flag

### ISSUE THREAD

**Issue by RonnyLV** [Riscure/Rhme-2016#4]
Hi,

I had an issue with trying to accomplish Casino challenge's flag - I found the flag but i didn't know that it was it. It was quite misleading because it didn't include word "FLAG" as in other programs and I thought that it's just a sign that I am moving into the right direction. And of course for many more hours later when I thought I got stuck in the void of oblivion and started to feel how the depression and regret embraces and takes over my body I finally talked myself into looking into the write-ups and only then I understood that I had already accomplished the challenge long ago...

tl;dr
Could you please prepend the challenge's flag with a word "FLAG"?

Best Regards,
Ronalds
